pages/poi/[id]/details.vue

```diff
@@ -77,7 +77,7 @@ const pageTitle = computed(() => {
   if (!poi.value)
     return ''
 
-  return `${poi.value.properties.classe} - ${poi.value.properties.name}`
+  return poi.value.properties.classe ? `${poi.value.properties.classe} - ${poi.value.properties.name}` : poi.value.properties.name
```

Comment:
`classe` is not a standard attribute provided by the API and will not be provided by the Elasa API.

Reply:
does `editorial.class_label_details.fr` is fine ?

Reply:
editorial.class_label_details.fr || editorial.class_label.fr